contrib/completions/bash/buildah

```diff
@@ -437,7 +438,7 @@ return 1
      esac
  }
 
- _buildah_umount() {
+ _buildah_unmount() {
      _buildah_umount $@
```

Comment:
That would create a recursion problem in the shell function.  To be fair, this hunk really belongs in its own PR.  I'll move it to one.